jdaviz/configs/cubeviz/plugins/tests/test_tools.py

```diff
@@ -87,10 +87,10 @@ def test_spectrum_at_spaxel_altkey_true(cubeviz_helper, spectrum1d_cube):
     assert flux_viewer.slice == 1
     label_mouseover = cubeviz_helper.app.session.application._tools['g-coords-info']
     label_mouseover._viewer_mouse_event(flux_viewer,
-                                        {'event': 'mousemove', 'domain': {'x': 1, 'y': 1}})
-    assert label_mouseover.as_text() == ('Pixel x=01.0 y=01.0 Value +1.30000e+01 Jy',
-                                         'World 13h39m59.9461s +27d00m00.7200s (ICRS)',
-                                         '204.9997755344 27.0001999998 (deg)')
+                                        {'event': 'mousemove', 'domain': {'x': 2, 'y': 1}})
+    assert label_mouseover.as_text() == ('Pixel x=02.0 y=01.0 Value +1.40000e+01 Jy',
```

Comment:
Did you visually vet if the new result is actually correct?

Reply:
I visually inspected a different cube but not the one in this test.